container/container_windows.go

```diff
@@ -10,6 +10,10 @@ import (
 	containertypes "github.com/docker/docker/api/types/container"
 )
 
+const (
+	containerSecretMountPath = `C:\ProgramData\Docker\secrets`
```

Comment:
I'm not too familiar with standard paths for application related data on Windows, so yes, any input welcome for me as well.